compute_endpoint/globus_compute_endpoint/executors/high_throughput/container_sched.py

```diff
@@ -42,9 +42,6 @@ def naive_scheduler(
                 int(math.floor(ratio * max_workers)), q_sizes[q_type]
             )
 
-        # CLEANUP: Assign the difference here to any random worker. Should be small.
-        # log.debug("Temporary new worker map: {}".format(new_worker_map))
```

Comment:
Are you leaving this in to be able to uncomment when needed?

Reply:
I was removing it, actually.  It has been commented out for quite awhile, and I don't see the utility to it at all: if it's necessary for debugging (as opposed to working with customer setup), then the dev can add it manually and _not_ as part of the commit/PR.

That said, I probably should have just ignored this hunk entirely: when Yadu's in-process work lands, I'm given to understand that this section goes away.